src/flamenco/runtime/fd_runtime.c

```diff
@@ -1238,6 +1238,110 @@ fd_runtime_block_verify_ticks( fd_blockstore_t * blockstore,
   return FD_BLOCK_OK;
 }
 
+/* MAP_KEY_NULL cannot be all 0s because it is the address of the Solana transfer system program */
+static fd_acct_addr_t fd_acct_addr_null = {.b={0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF}};
+
+#define MAP_NAME              fd_txn_writes
+#define MAP_KEY_T             fd_acct_addr_t
+#define MAP_T                 fd_txn_writes_t
+#define MAP_HASH_T            ulong
+#define MAP_KEY_NULL          (fd_acct_addr_null)
+#define MAP_KEY_EQUAL(k0,k1)  (0==memcmp((k0).b,(k1).b,32))
+#define MAP_KEY_HASH(key)     fd_hash( FD_TXN_CONFLICT_MAP_SEED, key.b, 32 )
+#define MAP_KEY_INVAL(k)      (0==memcmp(&fd_acct_addr_null, (k).b, 32))
+#define MAP_KEY_EQUAL_IS_SLOW 0
+#define MAP_MEMOIZE           0
+
+#include "../../util/tmpl/fd_map_dynamic.c"
+
+int
+fd_runtime_microblock_verify_read_write_conflicts( fd_txn_p_t *      txns,
+                                                   ulong             txn_cnt,
+                                                   fd_txn_writes_t * acct_map,
+                                                   fd_funk_t *       funk,
+                                                   fd_funk_txn_t *   funk_txn,
+                                                   fd_acct_addr_t ** conflict_opt ) {
```

Comment:
the lifetime looks suspicious. it's tied to either the txn or the acct lookup. can you just memcpy into an `fd_acct_addr_t *`?

Reply:
alternatively, you can document the lifetime semantics of the returned value